newIDE/app/public/External/jsfx/jsfx-main.js

```diff
@@ -0,0 +1,102 @@
+import {
+  createPathEditorHeader
+} from '../Utils/pathEditor.js';
+
+const electron = require('electron');
+const ipcRenderer = electron.ipcRenderer;
+const fs = require('fs');
+const remote = electron.remote;
+
+let editorContentDocument,
+  jsfx = null;
+
+const loadMetaData = metaData => {
+  jsfx.CurrentParams = metaData;
+  jsfx.UpdateCurrentView();
+  jsfx.PlayCurrent();
+};
+
+const closeWindow = () => {
+  remote.getCurrentWindow().close();
+};
+
+const saveSoundEffect = (pathEditor) => {
+  jsfx.UpdateDownloadLink(); //Update base64 data
+  let rawData = editorContentDocument.getElementById('download').href; //store data
```

Comment:
nitpicking: use const with a single assignment:

```
const rawData = editorContentDocument.getElementById('download').href.replace(/^data:audio\/wav;base64,/, '');
```

Prettier will help making this readable on 2 lines ;)
The reason is that using const allow to quickly see that this variable won't ever be re-assigned. 99% of the case you can use const.

Reply:
I was a bit worried it might not be readable initially, but you are right - prettify will make it readable!